src/objects/diff.ts

```diff
@@ -12,7 +12,7 @@ export const diff = (a: Hash, b: Hash) => {
   const aKeys = keys(a);
   const bKeys = keys(b);
   aKeys.forEach((key) => {
-    if (!bKeys.includes(key)) {
+    if (b[key] === undefined) {
```

Comment:
@mannan0819 Going from `.includes` to a direct check for a missing value is a good choice. Thank you.